sdk/identity/Azure.Identity/src/AzurePowerShellCredential.cs

```diff
@@ -144,6 +144,11 @@ private static void CheckForErrors(string output)
             {
                 throw new CredentialUnavailableException(AzurePowerShellModuleNotInstalledError);
             }
+
+            if (output.IndexOf("Microsoft.Azure.Commands.Profile.Models.PSAccessToken", StringComparison.OrdinalIgnoreCase) <= 0)
```

Comment:
Should this just be < 0? Is it possible that the output starts with "Microsoft.Azure.Commands.Profile.Models.PSAccessToken" or no?

Reply:
I think it would not be at index 0, but I agree it should be <0